bundle-workflow/tests/test_perf_workflow/data/test_manifest.yaml

```diff
@@ -0,0 +1,63 @@
+build:
```

Comment:
This is not test-manifest.yaml. Its bundle-manifest.yml, Rename it to avoid confusion. Here is the [test-manifest.yml](https://github.com/opensearch-project/opensearch-build/blob/main/bundle-workflow/src/test_workflow/config/test_manifest.yml)

Reply:
I'll address it in the next PR. Thanks

Reply:
Addressed in https://github.com/opensearch-project/opensearch-build/pull/404